package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "cht-conf-test-harness",
-  "version": "1.0.1",
+  "version": "4.0.2",
```

```suggestion
  "version": "4.0.1",
```

Let's keep it to 4.0.1 since that's the available tag and hopefully semantic-release updates it to 4.0.2 and releases.

---

Sure, fixed